packages/components/src/Actions/utils.js

```diff
@@ -0,0 +1,27 @@
+import omit from 'lodash/omit';
+
+const IGNORED_PROPS = [
+	'bsStyle',
+	'inProgress',
+	'disabled',
+	'hideLabel',
+	'link',
+	'onMouseDown',
+	'tooltipPlacement',
+	'tooltip',
+	'tooltipLabel',
+	'available',
+];
+
+export default function getRClick(action) {
```

Comment:
Why is it called RClick if it doesn't contain any specific logic related to right click?